apps/aecore/src/aec_governance.erl

```diff
@@ -33,6 +34,8 @@
 -define(TIMESTAMP_MEDIAN_BLOCKS, 11).
 -define(EXPECTED_BLOCK_MINE_RATE, 3 * 60 * 1000). %% 60secs * 1000ms * 3 = 180000msecs
 -define(BLOCK_MINE_REWARD, 10000000000000000000).
+-define(BLOCK_GAS_LIMIT, 8000000). %% Taken from Ethereum.
```

Comment:
Since we have 3 seconds between blocks rather than ~15 I guess a smaller value would make sense...

Reply:
Ok, I will use smaller number.